integration_tests/structs_02.py

```diff
@@ -0,0 +1,15 @@
+from ltypes import i32, f32, dataclass, CPtr, Pointer, c_p_pointer
+
+@dataclass
+class A:
+    x: i32
+    y: f32
+
+@ccallable
```

Comment:
@certik How will we support `ccallable` in CPython?

Reply:
I think its not supported in `LPython` as well. What do we mean by `ccallable`? Anything that can be called from `C` code?

Reply:
Yes. LPython supports it, it works with both LLVM and C. It can't work with CPython, since we can't build the C library that expects this function. The only way to support it from CPython would be to somehow expose LPython as a Python library and use it to generate C code, and then link it, etc. Too complex, we can maybe do that later, but for now we do not support `@ccallable` in CPython.

Reply:
I get the following error,

```bash
semantic error: Decorator: ccallable is not supported
  --> integration_tests/structs_02.py:10:1 - 17:12
   |
10 |    def f(a: CPtr) -> None:
   |    ^^^^^^^^^^^^^^^^^^^^^^^...
...
   |
17 |        y = a2.y
   | ...^^^^^^^^^^^^ 


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that needs to be fixed).
```

Reply:
Note the update - https://github.com/lcompilers/lpython/pull/614#discussion_r899118569